src/plugins/intel_cpu/src/nodes/kernels/scaled_attn/common.hpp

```diff
@@ -249,6 +307,11 @@ static constexpr size_t vec_len_f32_neon = vec_len_neon / sizeof(float);
         float32x4_t vec_f32 = vcvtq_f32_u32(vmovl_u16(vec_bf16));
         return vec_f32;
     }
+    inline float32x4_t __vld1q_f32(const ov::float16* a) {
```

Comment:
Why do we need changes in ARM code-path?

Reply:
It is to fix a compilation error. 
`scale_add2_reduce_max` 's template `T` can be `float16` and it calls `__vld1q_f32` in arm codes.